crates/aptos/src/governance/mod.rs

```diff
@@ -671,6 +675,9 @@ pub struct CompileScriptFunction {
 
     #[clap(flatten)]
     pub(crate) framework_package_args: FrameworkPackageArgs,
+
+    #[clap(long)]
+    pub(crate) bytecode_version: Option<u32>,
```

Comment:
@gregnazario please double check if this is correct. Initially I tried to add a move_options here but that caused the following test to fail:
```
---- test::tests::ensure_every_command_args_work stdout ----
Global logger has already been set
2022-12-16T20:44:22.121816Z [test::tests::ensure_every_command_args_work] ERROR crates/aptos-logger/src/logger.rs:47 Global logger has already been set
thread 'test::tests::ensure_every_command_args_work' panicked at 'Command generate-admin-write-set: Argument names must be unique, but 'skip-fetch-latest-git-deps' is in use by more than one argument or group', /home/vgao/.cargo/registry/src/github.com-1ecc6299db9ec823/clap-3.2.17/src/builder/debug_asserts.rs:100:9
```